Design system day speakers (#3192)
alphagov/govuk-design-system: src/community/design-system-day/speaker-information.md

---
title: Speaker and panelist information for Design System Day 2023
description: Speaker and panelist biographies and photographs 
section: Community
theme: Events and workshops
layout: layout-pane.njk
order: 13
---

<img src="/community/images/dsd23-announcement-banner.svg" alt="Design System Day 2023 logo" class="app-image--no-border" loading="lazy">

{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
{% from "_image-card.njk" import imageCard %}

{{ govukSummaryList({
  classes: "govuk-!-margin-top-3 govuk-!-margin-bottom-9",
  rows: [
    {
      key: {
        text: "Dates"
      },
      value: {
        text: "10 and 11 October 2023"
      }
    },
    {
      key: {
        text: "Location"
      },
      value: {
        html: '<a href="https://dynamicearth.org.uk/plan-your-visit/getting-here/">Dynamic Earth</a>, Edinburgh, and online'
      }
    }
  ]
}) }}

## Speakers 

Our speakers come from organisations across the UK and international public sector. We're joined by folks working on design systems from notable companies too.

## Day 1 speakers

### Kelly Lee (GOV.UK Design System team)

> Kelly Lee is a delivery manager on the GOV.UK Design System team. Before joining the Government Digital Service, she worked at Jobcentre Plus in a variety of roles including business coach, performance team leader and financial assessor.

Reviewer: Everyone else only has first name.

```suggestion
Kelly is a delivery manager on the GOV.UK Design System team. Before joining the Government Digital Service, she worked at Jobcentre Plus in a variety of roles including business coach, performance team leader and financial assessor.
```

Reviewer 2: I disagree, @calvin-lau-sig7. You only need to capitalise the job title when it's a proper noun, like on a business card or in an email signature. The rest of the time it's fine without capitals. 

`Kelly Lee is Delivery Manager on the GOV.‌UK Design System` would be OK but it also makes it sound like she's the only delivery manager.

He is currently a Community Designer at Government Digital Service, a department in the UK Government. He works on creating the right conditions for a community to thrive around the ever-popular GOV.UK Design System.

### Rachel Male (Scottish Governement)
Rachel is User Researcher and works in the Social Security Directorate for the Scottish Government. Rachel helps design disability and low income benefits for the people of Scotland and currently leads research operations focusing on operationalising their user research. Rachel has a passion for ensuring user research is effective, efficient and informative.

### Samantha Ernstzen (Scottish Governement)
To be confirmed

### Jennifer I'Anson (Scottish Governement)
To be confirmed

### GOV.UK Design System team 
The <a href="/design-system-team/" class="govuk-link">GOV.UK Design System team</a> is responsible for the implementation and maintenance of styles, components and patterns on the GOV.UK Design System. 

### Deborah Dada (Storelab)

> Deborah is a Product Designer with a strong focus on crafting meaningful user experiences and simplifying complex business solutions. Deborah brings a wealth of hands-on experience in UX principles and creating Design Systems with expertise that spans research, design systems and success metrics, underpinned by a design thinking approach. Deborah is currently a Product Designer at StoreLab and contributes to enhancing user satisfaction.

Reviewer: ```suggestion
Deborah is a Product Designer with a strong focus on crafting meaningful user experiences and simplifying complex business solutions. Deborah brings a wealth of hands-on experience in UX principles and creating design systems with expertise that spans research, design systems and success metrics, underpinned by a design thinking approach. Deborah is currently a Product Designer at StoreLab and contributes to enhancing user satisfaction.
```

### Caroline Jarrett (Effortmark)

> Caroline is the forms specialist. She advises organisations on how to make forms easier, improve websites and business processes. Her work spans all the types of design that will improve forms. Caroline is a sought-after speaker, trainer and author. Her latest book is Surveys that work: a practical guide for designing and running better surveys.

Reviewer: ```suggestion
Caroline is the Forms Specialist. She advises organisations on how to make forms easier, improve websites and business processes. Her work spans all the types of design that will improve forms. Caroline is a sought-after speaker, trainer and author. Her latest book is 'Surveys that work: a practical guide for designing and running better surveys'.
```

> ### Frankie Roberto (Department for Education)

Reviewer 3: Sorry - having seen the other bios I realised I should make it clear that I'm there as a freelancer not representing the department I’m contracted to.

```suggestion
### Frankie Roberto (Independent)
```

### Ellis Capon (Just Eat)
To be confirmed

### Janice Hannaway (ResearchU)
Janice is an empathic leader, a trauma-informed user researcher, therapist, and coach with experience working with people who have experienced the effects of trauma. She champions the psychological safety of user-centered design teams and highlights the potential harm from research and design on users. A co-founder of ResearchU Training Academy, she delivers specialist trauma-informed training

### Kate Every (Mastek)
Kate is a Lead Service Designer at Mastek, specialising in inclusive and ethical design and delivery. She led on accessibility, inclusive and equitable design for the UK's COVID-19 testing roll-out at NHS Digital. Heading up Mastek’s Service Design practice, she champions inclusive, trauma-informed approaches to research and design.  

### Helen Baron (The Emily Davison Centre)

> Helen leads the Move-On team and is centre manager at the Emily Davison Centre supporting domestic abuse survivors to access work, education, volunteering and training. Adopting a trauma informed approach within the service has enabled the team to see great leaps in confidence and self-esteem of the women they work with. 

Reviewer: ```suggestion
Helen leads the Move-On team and is Centre Manager at the Emily Davison Centre supporting domestic abuse survivors to access work, education, volunteering and training. Adopting a trauma informed approach within the service has enabled the team to see great leaps in confidence and self-esteem of the women they work with. 
```

### Tash Willcocks (TPXimpact)
Tash has spent over 25 years shaping future change-makers. First as Programme Leader of BA and MA Graphics at Salford University, then as Director of Masters at Hyper Island, working with the BBC, UN, NHS, Unilever and Adidas. After joining Snook, Tash created the learning design hub and launched the Snook Design Academy. Her most recent venture is joining TPXImpact to design and build their Design Academy.

### David Cox (GOV.UK Design System team)
To be confirmed

## Day 2 speakers

### Oliver Binns (GOV.UK One Login)
Oliver is a software engineer from Yorkshire with a master’s degree in Computer Science. He has worked on mobile apps for airlines, healthcare, and government, with a focus on iOS development. 

### Chris Choy (GOV.UK One Login)

> Chris is a UX design lead with a Computer Science degree. He worked as an SAP consultant before designing and delivering digital platforms and mobile apps, with a focus on experience design.

Reviewer: ```suggestion
Chris is a UX Design Lead with a Computer Science degree. He worked as an SAP consultant before designing and delivering digital platforms and mobile apps, with a focus on experience design.
```

 
### Helena Trippe (GOV.UK One Login)
To be confirmed 

> ### Frankie Roberto (Department for Education)

Reviewer 3: ```suggestion
### Frankie Roberto (Independent)
```

> Frankie Roberto is a lead interaction designer who has worked on several services across different government departments, including 'Apply for teacher training' and 'Ethnicity facts and figures'. 

Reviewer: ```suggestion
Frankie Roberto is a Lead Interaction Designer who has worked on several services across different government departments, including 'Apply for teacher training' and 'Ethnicity facts and figures'. 
```

      
He's also an experienced contributor to the design community and the GOV.UK Design System. 

### Katharine Beer (Department of Education)
Katharine Beer from the Department for Education is dedicated to crafting accessible and compassionate user experiences. 

### Abigail and Chloe Baldwin (Buttercrumble)
Twin sisters, Abigail and Chloe Baldwin founded the award-winning creative studio, Buttercrumble. Their specialist blend of design and strategy helps community-sensitive brands become the go-to choice for their audience, and authority in their industry. This is something they've helped to achieve for clients such as John Lewis and Mamas & Papas.  

### Sacha Zarb (GOV.UK Pay)

> Sacha is a Product Manager on GOV.UK Pay, working on improving the experience for the 700 services that use Pay. GOV.UK Pay is currently building new features like recurring card payments, mobile wallets and soon Open Banking to make it the default for payments in government. 

Reviewer 4: ```suggestion
Sacha is a Product Manager on GOV.UK Pay, working on improving the experience for the 700 services that use Pay. GOV.UK Pay is currently building new features like recurring card payments, mobile wallets and soon open banking to make it the default for payments in government. 
```

### Simone di Fresco (DOS Design)
Simone is a Design System specialist and facilitator at DOS Design which is working collaboratively in design with Italy’s Ministry of Economy and Finance. Beyond UX design, Simone is a facilitator in design and research workshops, emphasising co-creation and user-centricity.  

### Laura Smith (Unboxed)

> Laura is a lead designer at Unboxed, specialising in content. With particular experience in digital health, Laura has a keen interest in the importance of language and words in helping people access healthcare services. 

Reviewer: ```suggestion
Laura is a Lead Designer at Unboxed, specialising in content. With particular experience in digital health, Laura has a keen interest in the importance of language and words in helping people access healthcare services. 
```

### Sylvie Abookire (Independent)
Sylvie is a Service Designer specialising in public health, and a Coach in transitions and life design. Sylvie was formerly a Civic Designer at The Lab @ DC, an applied research team in the Washington DC Government. Some of her previous projects focused on housing stability and emergency responses to mental health crises.

### Daniel Fontaneda (Bumble)
After working as a product designer at Vodafone Group and Amazon, Daniel now leads the Design System at Bumble Inc. within a complex environment where the Design System supports 3 products: Bumble Date, Badoo and BFF (Bumble for Friends). Daniel’s role is to lead the definition and implementation of the product. 

### Ciandelle Hughes (GOV.UK Design System)
Ciandelle is an Interaction Designer on the GOV.UK Design System. She is an advocate for accessible design as well as a strong believer in human-based design. 

### Ashley Watson-Nolan (Just Eat)

> Ashley is a Principal Engineer at Just Eat Takeaway with over 15 years experience in web development and is a fan of all things visual. He specialises in component-driven development and front-end architecture and is passionate about emerging front-end technologies. He hacks about with CSS and JavaScript, blogs at ashleynolan.co.uk and is @WelshAsh_ on X. 

Reviewer: ```suggestion
Ashley is a Principal Engineer at Just Eat Takeaway with over 15 years experience in web development and is a fan of all things visual. He specialises in component-driven development and front-end architecture and is passionate about emerging frontend technologies. He hacks about with CSS and JavaScript, blogs at [ashleynolan.co.uk](https://ashleynolan.co.uk/) and is [@WelshAsh_](https://twitter.com/WelshAsh_) on X. 
```

### Debra Churchill (Ontario Public Service)
Debra’s career has focused on human services. Before leaving New Zealand for Toronto, Canada, Debra worked in operational and policy roles. Since 2015 Debra has used service design methodology to co-create and re-design programs and services. Debra has been part of transformation projects including digital transformation, developmental services transformation, and transfer payment modernisation. 

### Gordon Guthrie (Scottish Government)
Gordon is a Research Fellow at the Scottish Government on the First Ministers Digital Fellowship Programme. His professional life started as a code monkey in the private sector and has ranged from Chief Technical Architect at if.com, scalability guru at bet365 and ending up as a Silicon Valley VP Engineering. He has worked in public policy in London, Belfast and Scotland.
 
### Jane Martin (Government Digital Service)

> Jane is a senior service designer at the Government Digital Service. Before this she worked for 20 years as a theatre producer and specialist in arts-based methods in international development. She's worked in over 20 countries including 10 years in Cambodia, working with some of the most marginalised groups in the country. 

Reviewer: ```suggestion
Jane is a Senior Service Designer at the Government Digital Service. Before this she worked for 20 years as a theatre producer and specialist in arts-based methods in international development. She's worked in over 20 countries including 10 years in Cambodia, working with some of the most marginalised groups in the country. 
```

### Betty Mwema (Government Digital Service)

> Betty Mwema is a senior service designer in the GOV.UK Forms team at the Government Digital Service. She has worked in the social and private sectors for 7 years, with a strong emphasis on inclusivity and ethics. Betty enjoys creating and improving design methods, to ensure that we equitably and inclusively meet the needs of users.

Reviewer: ```suggestion
Betty Mwema is a Senior Service Designer in the GOV.UK Forms team at the Government Digital Service. She has worked in the social and private sectors for 7 years, with a strong emphasis on inclusivity and ethics. Betty enjoys creating and improving design methods, to ensure that we equitably and inclusively meet the needs of users.
```
